app/models/organization.rb

```diff
@@ -46,8 +46,9 @@ def slugify
     self.slug = "#{github_id} #{title}".parameterize
   end
 
-  def create_organization_webhook(webhook_url)
-    webhook = github_organization.create_organization_webhook(config: { url: webhook_url })
+  def create_organization_webhook(webhook_url, client = nil)
```

Comment:
I added an optional parameter `client` here just in case we randomly picked other admins who don't have enough token scopes to manipulate organization webhook.